api/src/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# node:18.12.1-alpine3.17
+#FROM node@sha256:b375b98d1dcd56f5783efdd80a4d6ff5a0d6f3ce7921ec99c17851db6cba2a93
+
+# node:18.12.1-slim
+FROM node@sha256:0c3ea57b6c560f83120801e222691d9bd187c605605185810752a19225b5e4d9
+
+RUN apt-get update && apt-get install -y curl
```

Comment:
@joseph-bisonai Are we using curl somewhere?

Reply:
Curl some times very useful. When I communicate with other pod with local domain

Reply:
When I want to check pod to pod communication with local kube-domain, 
Since we are using api many,
The local domain address looks like this.
http://orakl-api.orakl.svc.cluster.local:3030/api/v1

then any other pod you can get result call with curl

`curl http://orakl-api.orakl.svc.cluster.local:3030/api/v1`

fetcher also.~

Reply:
Got it! Thank you!